dev/io.openliberty.java.internal_fat/src-reference/java18/io/openliberty/java/internal/TestService.java

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.java.internal;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Iterator;
+import java.util.random.RandomGenerator;
+import java.util.random.RandomGeneratorFactory;
+import java.util.stream.IntStream;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/")
+@ApplicationScoped
+public class TestService {
+
+    private StringWriter sw = new StringWriter();
+
+    @GET
+    public String test() {
+        try {
+            log(">>> ENTER");
+            doTest();
+            log("<<< EXIT SUCCESSFUL");
+        } catch (Exception e) {
+            e.printStackTrace(System.out);
+            e.printStackTrace(new PrintWriter(sw));
+            log("<<< EXIT FAILED");
+        }
+        String result = sw.toString();
+        sw = new StringWriter();
+        return result;
+    }
+
+    private void doTest() throws Exception {
```

Comment:
Please remove the Java 17 specific tests from this file (similar to open-liberty-misc request).

Reply:
Do we want tests to be additive, since capabilities added by previous GA'd JDKs should still be present (unless they've been deprecated)?

Reply:
Good point Joe, nevermind.